streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsPartitionAssignor.java

```diff
@@ -185,24 +187,25 @@ public String toString() {
     public void configure(final Map<String, ?> configs) {
         final AssignorConfiguration assignorConfiguration = new AssignorConfiguration(configs);
 
-        logPrefix = assignorConfiguration.logPrefix();
+        logPrefix = assignorConfiguration.getLogPrefix();
```

Comment:
Why do we use `getLogPrefix()` -- our code style _always_ omit the `get` prefix -- should be just `logPrefix()` (Similar below for other methods)

Reply:
I guess I thought there was a reason `getXX` was used in this class and figured all the methods should conform. Idk. I'll remove the `get`